sabnzbd/__init__.py

```diff
@@ -179,6 +179,7 @@ def sig_handler(signum=None, frame=None):
     if signum is not None:
         logging.warning(T("Signal %s caught, saving and exiting..."), signum)
     try:
+        Downloader.do.set_paused_state(True)
```

Comment:
This is done to make the article queue get purged and stop it from adding new articles to the queue.